src/xmerl_dsig.erl

```diff
@@ -221,19 +220,20 @@ do_find_content([Head | PTail], Parents, Elem) ->
 %% the MinaMeddelanden API.
 %% It will now verify all signatures inside the SOAP structure.
 -spec verify(Element :: #xmlElement{},
-             Fingerprints :: [#'OTPCertificate'{}] | [fingerprint()] | any) ->
-          ok | {error, bad_digest | bad_signature | cert_not_accepted}.
-verify(Element, Fingerprints) ->
+             CaCerts :: [#'OTPCertificate'{}] | [binary()] | any) ->
+          ok | {error, bad_digest | bad_signature} |
+          {error, {bad_cert, atom()}}.
```

Comment:
`{error, {bad_cert, public_key:bad_cert_reason()}}`

Reply:
that worked.